projects/schematics/README.md

```diff
@@ -59,44 +59,76 @@ The best way to test an unpublished schematic is to publish it to a local npm re
-- adding a comment above the Ngrx action in which we changed parameters
+The following are examples of how you might add a comment:
+
+- If you removed an API method, you could add a comment above the removed method that suggests which method can be used instead.
+- If you changed the parameters of an NgRx action, you could add a comment above the action where the parameters were changed.
 
 ### Component deprecation
 
-Similar to [constructor deprecation](#Constructor-deprecation), `projects/schematics/src/migrations/2_0/component-deprecations.ts` performs the component migration tasks, for both component _*.ts_ and _HTML_ templates. Usually, a developer does not need to touch this file, and they should rather describe the component deprecation in `projects/schematics/src/migrations/2_0/component-deprecations-data.ts`. The constant `COMPONENT_DEPRECATION_DATA` describes the deprecated components.
+Similar to constructor deprecation, `projects/schematics/src/migrations/3_0/component-deprecations.ts` performs component migration tasks, for both component `*.ts` and `HTML` templates. Usually, a developer does not need to touch this file, and instead should describe component deprecations in `projects/schematics/src/migrations/3_0/component-deprecations-data.ts`. The `COMPONENT_DEPRECATION_DATA` constant describes the deprecated components.
 
 ### CSS
 
-To handle CSS changes, we are printing a link to the CSS docs, where customers can look up which CSS selectors have changed between Spartacus versions. For this reason, if making a change to a CSS selector, please update this docs. (link to follow).
+To handle CSS changes, we print a link to the CSS migration documentation, where customers can look up which CSS selectors have changed in the new version of Spartacus. If you are making a change to a CSS selector, simply update the relevant documentation (such as, [Changes to Styles in 3.0](https://sap.github.io/spartacus-docs/css-changes-in-version-3/)).
+
+### Adding a Migration
+
+The following is an example flow for adding a migration:
+
+- Check whether any of the changed files are exported in the public API. If no, then no further action is required.
+- Check whether any of the changes you have made are breaking changes. If not, no further action is required. For more information, see [Maintaining Public APIs]({{ site.baseurl }}{% link _pages/contributing/breaking-changes.md %}).
+- For every breaking change, you must do the following:
+  - Document the breaking change by updating the corresponding migration doc file (such as `docs/migration/3_0.md`), and if necessary, ensure that code comments have been added.
+  - Build automation tasks, as described in the [Validations](#validations), [Constructor Deprecation](#constructor-deprecation), and [Component Deprecation](#component-deprecation)) sections, above.
+  - Test the added migrations by running tests, trying to migrate an example app, and so on.
```

Comment:
Maybe we can link the part where it's explained how:
- to write / run unit tests
- to migrate an example app